util/cron/test-gasnet.numa.bash

```diff
@@ -6,6 +6,8 @@ CWD=$(cd $(dirname $0) ; pwd)
 source $CWD/common-gasnet.bash
 source $CWD/common-numa.bash
 
+export CHPL_NIGHTLY_TEST_DIRS="release/examples/hello*.chpl localeModels"
```

Comment:
Maybe stuff this into common-numa instead?

Reply:
I started to do that and then realized it would get picked up by `test-perf.chapcs.numa.bash`, which we don't want.  We could unset it afterwards, but that seems clunky.